.changeset/flat-sheep-own.md

```diff
@@ -0,0 +1,6 @@
+---
+"@razorpay/i18nify-js": minor
+"@razorpay/i18nify-react": minor
+---
+
+Feat: Mono-repo setup added for i18nify and i18nify plugins
```

Comment:
it doesn't matter for consumers because its an infra level change. Ideally this should be a `patch` release because you just want to publish a lib from this setup once to sanity that the infra is working

Reply:
+1